typings/objection/index.d.ts

```diff
@@ -597,6 +597,7 @@ declare namespace Objection {
 
   interface WithMethod<QB extends AnyQueryBuilder> {
     (alias: string, expr: CallbackVoid<QB> | AnyQueryBuilder | Raw): QB;
+    (alias: string, columns: string[], expr: CallbackVoid<QB> | AnyQueryBuilder | Raw): QB;
```

Comment:
I'm no TS specialist and don't personally use typings, but `columns` is optional, and this doesn't seem to be reflected here? Please make sure the typings work for both patterns, with and without columns.